app/views/content/funding-and-support/salaried-teacher-training.md

```diff
@@ -110,7 +110,9 @@ Some salaried courses award QTS only. You may have the option to do a salaried c
 
 You can [find postgraduate teacher training](https://find-teacher-training-courses.service.gov.uk/) and filter by salaried courses. 
 
-It’s usually wise to apply for non-salaried courses as well to increase your chances of getting a place on a course. 
+It’s usually wise to apply for non-salaried courses as well to increase your chances of getting a place on a course.
```

Comment:
I know this is in prod already but I wonder whether it is useful to talk about `fee-paying courses` rather than `non-salaried courses` to distinguish them more - we don't really use `non-salaried` on the site (except in a couple of minor places)?

@emmaachesongray @Sarah-DfE